R/pkg/R/mllib_tree.R

```diff
@@ -409,7 +413,7 @@ setMethod("spark.randomForest", signature(data = "SparkDataFrame", formula = "fo
                    maxDepth = 5, maxBins = 32, numTrees = 20, impurity = NULL,
                    featureSubsetStrategy = "auto", seed = NULL, subsamplingRate = 1.0,
                    minInstancesPerNode = 1, minInfoGain = 0.0, checkpointInterval = 10,
-                   maxMemoryInMB = 256, cacheNodeIds = FALSE) {
+                   maxMemoryInMB = 256, cacheNodeIds = FALSE, handleInvalid = "error") {
```

Comment:
use match.arg(), and then no need to as.character(handleInvalid)

also, perhaps handleInvalid is a bit generic? maybe something that says it has to do with labels? or label string indexing?